.env

```diff
@@ -34,3 +34,4 @@ STICKERS_TEST_ENABLED=1
 LOCAL_PAIRING_ENABLED=1
 TEST_STATEOFUS=1
 FAST_CREATE_COMMUNITY_ENABLED=1
+FLAG_BRIDGE_TOKEN_ENABLED=1
```

Comment:
do we want this?

Reply:
Not sure. I couldn't access to Bridge option in develop without this. WDYT?

Reply:
For local development it's fine but if you look in options there's a feature flag toggles page